### DISCUSSION TITLE
coherence doubt

### DISCUSSION THREAD

**Discussion by MocioVileda** [insarlab/MintPy#987] [Q&A]
Hi everyone!
I'm processing HyP3 derived interferograms in MintPy. How is it possible that, despite getting coherence values higher than 0 for my AOI, as shown in the first image of the 4 interferograms, I then get nan values for the temporal coherence and coherence matrix?
![coherence](https://user-images.githubusercontent.com/129189498/230181490-00ae4e24-2898-4bb4-a877-e7bf47748c11.jpg)
<img width="206" alt="NaN_coherence" src="https://user-images.githubusercontent.com/129189498/230181516-8ab54a67-ae64-41fc-bac4-4d7557fccbf3.PNG">

I tried to recompute it, but I always get the same result...
Thanks!

**Comment by yunjunz** [comment]
I would suggest checking the water mask file, and choosing a reference point that has high coherence.
